### ISSUE TITLE
adding Admin Panel

### ISSUE THREAD

**Issue by shakesBeardZ** [thedevs-network/kutt#157]
I'm thinking about adding an admin area to the project.
A simple dashboard to contain a few important statistics about the use of the website and all the users.
give your suggestions and what do you think about it.

**Comment by trgwii** [comment]
Excellent, go for it!

We will probably have more suggestions once you have made something cool for us to check out

**Comment by poeti8** [comment]
Sounds like a good idea, we already have a section in the settings page to ban links and it's only visible to the admin. Maybe can add some more actions and stats there.

**Comment by shakesBeardZ** [comment]
poeti8 so we need to decide what we gonna put in it, then we'll start building it
I suggest the following:

1. Managing all users
2. Stats about all created short links 

and it doesn't need to be on nextjs, it can use just Reactjs plus it can in a different repo and use the same API.

so I'm waiting for your technical suggestions.

**Comment by poeti8** [comment]
Yes, those are good for start. Also need to manage links there. 
I prefer it to be on the app itself, but of course we should provide the the API for people who want to use it in elsewhere.

**Comment by ahkui** [comment]
any update for admin panel ?

**Comment by EdisonJwa** [comment]
How about make site private ? I mean custom the url redirects when user enter the hostname and the homepage real, such as create the shortened urls for logined users only
And custom the default path for shorten urls?

**Comment by Max-hm** [comment]
Hi, Did an admin panel get created or is it still on the roadmap?

Manage users

See and manage all links

Housekeeping

**Comment by poeti8** [comment]
All of that can be done via the table, if you login as admin you can search through all links and delete or band them.

**Comment by Max-hm** [comment]
@poeti8 commented 20 hours ago
> All of that can be done via the table, if you login as admin you can search through all links and delete or band them.

Sorry poeti8 I am not sure I follow you. I am surely being a complete n00b and will face palm when you point our my ignorance for all to see :-(. I have set up Kutt and there doesn't seem to be an admin login that I can see. Each user seems to set up their own account that is visible to only them.
